new quickstart guides (#3060)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/index.md

---

> title: Sign users into your SPA using the redirect model

Reviewer: ```suggestion
title: Sign users in to your SPA using the redirect model
```

> excerpt: Configure your Okta org and your single-page app to use Okta’s redirect sign in.

Reviewer: ```suggestion
excerpt: Configure your Okta org and your single-page app to use Okta’s redirect sign-in flow.
```

layout: Guides
---

Add authentication with Okta's [redirect model](https://developer.okta.com/docs/concepts/redirect-vs-embedded/#redirect-authentication) to your single-page app. This example uses Okta as the user store.

> > **Note**: For server-side web apps, see [Sign users into your web app using the redirect model](/docs/guides/sign-into-web-app-redirect/) instead. To protect the API your SPA calls, see [Protect your API endpoints](/docs/guides/protect-your-api/).

Reviewer: ```suggestion
> **Note**: For server-side web apps, see [Sign users in to your web app using the redirect model](/docs/guides/sign-into-web-app-redirect/) instead. To protect the API your SPA calls, see [Protect your API endpoints](/docs/guides/protect-your-api/).
```

---

**Learning outcomes**

> * Create an integration representing your app in your Okta org.

Reviewer: ```suggestion
* Create an integration that represents your app in your Okta org.
```

* Add dependencies and configure your app to use Okta redirect authentication.

> * Add sign-in and sign-out.

Reviewer: ```suggestion
* Add sign-in and sign-out flows.
```

* Require authentication on protected routes.
* Get authenticated user info.

> * Make an HTTP call with the Access token.

Reviewer: ```suggestion
* Make an HTTP call with the access token.
```
